dspace-api/src/main/java/org/dspace/app/mediafilter/ImageMagickThumbnailFilter.java

```diff
@@ -132,12 +136,22 @@ public static File getImageFile(File f, int page) throws IOException, Interrupte
     	f2.deleteOnExit();
     	ConvertCmd cmd = new ConvertCmd();
 		IMOperation op = new IMOperation();
+		Info imageInfo = new Info(f.getAbsolutePath(),true);
+		// I'd rather set these in the class but can't figure out how to access them here if I do!!!
```

Comment:
Perhaps I understand the question after taking a closer look.  I recommend that you move the declaration for cmyk_profile  and srgb_profile before the static { ...} block and declare them as static.  Initialize them inside the static {} block.  You can then reference the initialized values in getImageFile()